sktm/jenkins.py

```diff
@@ -68,9 +68,9 @@ def __get_data_list(self, jobname, buildid, stepname, key):
             raise Exception("No results for build %d (%s)" %
                             (buildid, build.get_status()))
 
-        for (k, v) in build.get_resultset().iteritems():
-            if k == stepname:
-                value_list.append(v.__dict__[key])
+        for (key, value) in build.get_resultset().iteritems():
```

Comment:
There's already a `key` parameter of the function, so I'm afraid this change would break the build.

Reply:
What about the name `keyname`?

Reply:
Works for me, although I'd prefer it with an underscore (`key_name`). Or what occurred to me is `result_key`, since we are iterating over a set of results here, what do you think?

Reply:
Yeah. It is good. I will fix it now.

Reply:
Descriptive variables are the way to go so I also think that one is more suitable

Reply:
I fix it. Ah `pylint` does not create the warning for `k` variable. That is reason why I did not modify the `k` variable

Reply:
I think pylint doesn't complain about it is because `i`, `j` and `k` are acceptable names for iterators, so it doesn't check the fact that we aren't looping over a list here. However, pylint not complaining doesn't mean the code is clear and readable :)